### ISSUE TITLE
PyTorch RNN example: from scratch

### ISSUE THREAD

**Issue by xuyuji9000** [xuyuji9000/machine-learning-playground#70]
~Refer to this [article](https://jaketae.github.io/study/pytorch-rnn/).~

- [x] draw a draw.io diagram of the RNN model 

# Check the following concepts

1. - [x] logits
    An [article](https://www.geeksforgeeks.org/what-are-logits-what-is-the-difference-between-softmax-and-softmax-cross-entropy-with-logits/)

2. - [x] negative log likelihood
    An [article](https://medium.com/deeplearningmadeeasy/negative-log-likelihood-6bd79b55d8b6)

**Comment by xuyuji9000** [comment]
building the RNN model from scratch example added.

Closing this issue.
